libs/application/templates/children-residence-change/src/lib/utils.ts

```diff
@@ -28,6 +28,7 @@ export const extractAnswersFromApplication = (application: Application) => {
     selectedChildren: application.answers.selectChild as string[],
     selectedDuration: application.answers.selectDuration as string[],
     reason: application.answers.residenceChangeReason as string,
+    interview: application.answers.interview as 'yes' | 'no',
```

Comment:
Do we want to use the ValidAnswer type here?

Reply:
Thought about it, I want to go through our code as a whole and find a good place for shared types and more specifically type the answers object completly instead of doing it inline like this..